images/Makefile

```diff
@@ -25,9 +25,15 @@ lint:
 runtime-image: .buildx_builder
 	TEST=true scripts/build-image.sh cilium-runtime-dev images/runtime linux/amd64,linux/arm64 $(OUTPUT) "$$(cat .buildx_builder)" $(REGISTRIES)
 
+update-runtime-image:
```

Comment:
Please add more info in the commit, it's not clear to me what is the purpose of these scripts / makefile targets.

Reply:
This is meant to be documented in [`images/READE.md`](https://github.com/cilium/cilium/blob/ea59fd095d7c38f70a82a9498219f69f1f6a192c/images/README.md), have you had a look at it yet?